documentation/datamodel/datamodel-properties.asciidoc

```diff
@@ -299,7 +299,7 @@ class MyConverterFactory extends DefaultConverterFactory {
 }
 
 // Use the factory globally in the application
-Application.getCurrentApplication().setConverterFactory(
+UI.getCurrent().getSession().setConverterFactory(.setConverterFactory(
```

Comment:
There is one .setConverterFactory to much :)

Reply:
uuups, indeed Thanks! :)